js/main.js

```diff
@@ -25,5 +31,100 @@ function submitForm(e) {
   };
   data.entries.unshift(submission);
   $image.src = 'images/placeholder-image-square.jpg';
+  $ul.append(renderData(submission));
```

Comment:
small update, you want to use `prepend` here to add it to the front of the other children of the ul